identity-apps-core/apps/authentication-portal/src/main/webapp/emailOtp.jsp

```diff
@@ -32,6 +32,32 @@
 <%-- Branding Preferences --%>
 <jsp:directive.include file="includes/branding-preferences.jsp"/>
 
+<%!
+    private boolean isMultiAuthAvailable(String multiOptionURI) {
+        boolean isMultiAuthAvailable = true;
+        if (multiOptionURI == null || multiOptionURI.equals("null")) {
+            isMultiAuthAvailable = false;
+        } else {
+            int authenticatorIndex = multiOptionURI.indexOf("authenticators=");
+            if (authenticatorIndex == -1) {
+                isMultiAuthAvailable = false;
+            } else {
+                String authenticators = multiOptionURI.substring(authenticatorIndex + 15);
```

Comment:
Can you share a sample of a URI we are trying to extract the authenticators from?
Just want to know if 15 would work everytime.

Reply:
```
/authenticationendpoint/oauth2_login.do?authenticators=email-otp-authenticator%3ALOCAL%3Bsms-otp-authenticator%3ALOCAL&response_type=code&code_challenge_method=S256&type=oidc&client_id={{client-id}}...
```